deno.d.ts

```diff
@@ -0,0 +1,2254 @@
+// Copyright 2018-2019 the Deno authors. All rights reserved. MIT license.
+
+/// <reference no-default-lib="true" />
+/// <reference lib="esnext" />
+
+declare module "deno" {
```

Comment:
Remove this file

Reply:
@ry Oh, I'm sorry. I'm using this for types in VSCode